conanfile.py

```diff
@@ -1,87 +1,57 @@
 from conans import ConanFile, CMake, tools
 import os
-import shutil
-import re
+
 
 class MongoCxxConan(ConanFile):
     name = "mongo-cxx-driver"
-    version = "3.3.0"
+    version = "3.4.0"
     url = "http://github.com/bincrafters/conan-mongo-cxx-driver"
     description = "C++ Driver for MongoDB"
-    license = "https://github.com/mongodb/mongo-cxx-driver/blob/{0}/LICENSE".format(version)
-    settings =  "os", "compiler", "arch", "build_type"
-    options = {"shared": [True, False]}
-    default_options = {'shared': 'False'}
-    requires = 'mongo-c-driver/[~=1.9]@bincrafters/stable'
+    license = "https://github.com/mongodb/mongo-cxx-driver/blob/{0}/LICENSE".format(
```

Comment:
```suggestion
    license = "Apache-2.0"
```

Use the short format instead